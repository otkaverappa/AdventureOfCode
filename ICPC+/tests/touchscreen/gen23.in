19
quhqhytwtmppwbqsjwzxidf 10
wuddferqwlymuyiqruhhrjz
juvkuenbchegnkzfimdugce
wvzgesrebnznhfbkikqqzap
fibifhmjfkinoaesdsurbzu
oheezgfolprvuykeguotkra
fwszpvajjkqojwtajirefso
jakztvstapnbgjbqsliixgb
xpmqvtikeiwdetwztnnwtyy
lqlvbibchsqyoliyprwtxqr
wlejizjilavgbgkfncynqjw
qznnumfniqtuysdiffeoimy 10
qzcqptbllcaywmdjwweqwxw
fcblsmxqcytspovazyaauof
glbkbyjxfobqgnpzrpujfie
hjekjzyrhjstnsdmxrnpxcg
ytwsedalkmpuvqvoxehttzw
gzpvooyvnrrfwwijiuydrpa
goephyofgnubejzzyrsdpon
ywhybbpbheiwnimtocppgar
fykxebnrzlbzpcpsjtchgkt
bavrsbrjisthwuupviquxsl
pdesknegqfdhxfbipvbxrxt 10
gsbzoyraqvgkhvonmvrvsyy
aptdmmagvuhulilnyitiqry
ffuyamwbpmuuajxietqyeal
ryunimggtlasnpqogclafuz
nytfyxfjrddglrmulxudkjt
brymsdirsimygkdpdhvjuov
ozulegwahoyviddbvnovvyz
ndgkmcyjtadeejagrogowlu
ahhoedodqrlcdoaoharnhan
nteuicfccolqtqeyixkkcal
qjnhwwkjkfngnsnrueflzvr 10
xerjotjbksrtondazjphyhx
scbxpcsibyznntwqfrhyfxc
hwnxengbfkeczixilvjykcv
bhmkgthkdezckminuttwubv
emronbqicawpirwuwctyohj
bbeuayxvdmmtzqjjulfjtyi
nvkivzqectizthyrmkffjvq
uqcbbxajnxuvsvnzzhhycqx
vejblvycpqgrsftbsqxokud
juktupjtkqdnbaofpeliwrb
yjtvzeqdpmpkggwctmhgnhu 10
twzghxzivletkklpzxfkflg
heuonjvjeuklbimlfxpykbj
zbggnouvvapknyhzddjqgtc
szcjxmlhlopuddoybopnepu
duammggpyftkefxnqldkqia
qlphytwvwtvkfbswczpmeul
tnywzohaeurltkjqiclukop
gqoyeuupohnmjdvjhqculoe
bwiosschzvyzzutrkbgzmku
xtmzoxnsalaofvtpwonvpjo
iurpvdbpauebiboccbfshal 10
wwzlrruhzqzqlerdfmhgwkx
zmyefhgsdcsovllckbdbpis
upgbxdlweaxkheqziutypej
tpnzsehmyytbxymucmrovxe
vfaogrvrkmgjeksqksllqja
mnmfbkchfkhvqyqjlcqwiap
znjmybyypnnupxzyelizbgs
ktucqddhesqqquqpjeegvbf
tfsbgtkyfdtkvypcehvuxrl
oarsimvpfchjjcvhkacuwcj
agqxdnoqerhwbtsrauyjfdg 10
nnhrhfuqfciffvtvzmcyoxs
fyofjuiqjxzceeywjbeqgck
edyibogvvgkbpelhpjirpmq
oyrsozwadazmdnuzlajmrpz
bekjxcwnqugkkihlnhmbmbv
lmxwarlcsrmbrqzgivntgxc
tkjhmxijjxjfxatzsmovdgx
kpsziawkvivcwsngdkpljpe
jjqzewhbgwwfgybqujnyiil
onvfyfweqhxrltypzxnedlh
whslgattqhpxhwtmoajhoch 10
fdeshifqgpkrxlnqdwhbdfw
rtzaaneiuhooowvgekrwhej
yhhhkknhcggejvjtrqhfhga
pmngiwmtuuaeenrnrztxkqi
dhznoivrzheisaunxqodvdu
kwvgtgwclgdawnvnoftwxwr
mvhayefupdpvzvtbjbgjyvh
njmhglfzxaiaanhwellzkju
nswvduqkhbyqjlxilzrnzgu
vkhjjtsdimbdrvvcfxcxiov
rzukowqitascllwqukszcpv 10
gwuiucyrvznfnlxyfxqjkdi
bxsvxwmuejopfqnympltcxr
aeqsnbwvevqzsolotwcibvv
zjkncpktrqkkdngbrbtswhf
kccpkfkihtuuvjgrbwdnbst
elwxdshkfjouwugebzyxwkd
plbsyektjvpiakrlrdbnzvt
dyubxqncgafaakntvklfnxx
aoabbzzwezqfxgvzovfrwre
pmqcrdzqfpqhsrgovfedeny
gdtnupeahstllylceculvmt 10
ljasmfhmnkiahzehgyasjzf
lfjiqgulsflkustgfdqhkqn
uylwqvxxichqsnmfhsrtolm
utslacbqybbvsysrjvazpqo
vxiotwzfsuxdvcgnaiovcpq
vylyzaopvnxlgtlomhlrepa
rqkinoxdjxohwrxmoljarfn
ftutkzimsawaqkxtwukftbd
froqrhxgpqdkdeupwvoxlaw
gyrqemtjrlxjegjlxzohesc
wozknmmksldjrpcaipzrufc 10
fdcvmgnoednorzdduogdhvh
hdxgwrnzyqdtflitqljgekj
heznjgkrnpqvojjnhbsagfi
cvvleaxpjepznxjgnzyinht
cqvwqbbafzysdaqsjujijjs
swtqgiaamqxijaljgkjynjq
itktbwcvoyqghygjmxguggi
rmsbnikdqfqljmogcoxkvxr
ekqmgwtophjsurfyjkoutci
agzzquwhbiznrxghnqqfkhk
isxypscxuiytztsiwcijtfs 10
buitcsdodvldkegkaqiwpdo
azsklmrdpmniohlckjqqtuy
fwoqsgtgivbsgpjlexyolfc
qsngiicioyyerezzmuagars
lxplmdrqtldbvlddbdbhuih
wwbydeqvrnmdbpwtkhwlckq
hluivqqconfnslfpdsbiwtg
pdportyjaldkgwbkkqqzkbe
zgzaqvvykznysiybimhtudw
fpoxpyavxjuzbpvacwpcoyb
ripybjdzqtnpknqfkzakcrh 10
etduwruzicqjgzmfshvhriz
ztybfdumhorbmkxnuaddges
nyzigtrfsmgvtlphsfywtit
gcvlibsarqbzyusgogojzzz
grgenaojgteudiodzggafbt
npbbbbcchtiojieurzylcgz
hhfoimrdzgecjhelrauflcl
htlfgpohxxpnhbemdmrhwao
hrqbybgkkbxpimfpjdeykic
xnpqulqiskywibcsldpdnci
fnnlmytqxgfndqfllpmhapl 10
sbokgeoqjdeureqibwnqbgx
ntnbcpqnhtdtbhjsqnwnead
obbetdkqsffthulwefzxgjj
azwwocyrrbtvuwhlooqjxlu
trbuqkdzkfvjtahltjeodcx
qsnjbkwxdnzzdlddxkagdcq
qvzwleykxqxgthcrmrsmufp
zczcleebvactliswfktlgdp
zpgtbdyscctepyjsvluoyfi
ukvqydvpckyvobtgfxcmncx
fzicpghydtvttyqlaajxqks 10
zrpbdcdakeiotrxtuqopmof
xqhgnysfxjvynzdplhazcrw
wnpmcddhatqijocollllanq
qyxsacjyywnmaspjsjzcsoe
ibswnslfkldclhnkflztlsj
umuupmkvwnnuaifgutlxeum
qzyssjmdgyxcplmlkzaecim
kdfxclrqesowmaiqginiybw
llvnpzwdjbiifuzvyrmwgme
wsmlcmmyxxvlpvjsgmbptxq
qxkctqozoinzkaoizlgmadx 10
tjlwaevtxsdznvomjybkjdy
tdmzrmcrhoefqjajiuohhlx
vuwxhhafafrczggqpwuxnfz
ynupvbauxxqwezwjccdeblm
tcjnzyvzyivnfwqcvnsrruq
cwshavsoovzdwzyxyjsnqrd
soqnijhymdstfnotdlvihuj
hssziksldhespnbpbehxlul
ezwzhfuqmokoyvikapqugfv
vhzeekarqznsjuozhemxdhf
njxfffmadtbhznisfjfxsal 10
sjqegtnniykpdqbdtxgaytk
ryuyymajgjbmrwzfgzrxehz
hcypfwippjlohyqsgcwttuu
abusaybbiezxlyiaptofamw
usyqntliwocqocrsmxrmirw
lhpbmpnilhicvepdadhstll
nyjhmtyighxhuowezgpddts
ivxpntcygcjnofmwntvxpjt
wlbloegjmbgdqbiqikcvbjj
xwfvtwgpsrqfixltjozpedx
xlhziiiuigzdaxlrqdjvngj 10
gsxhhekgbxncgymaisbnuyy
nqdxosfxyxuigysmbqcdyar
akmbzibxwsdvgvadwazegyy
tbqvhqxyqhmrgvvftnkqvgs
aeszkzzksdbpktmkkvxdduy
kplvfidzihskttldnogcyao
lmlipoeoytbvakyzuhrgams
ezimbjoruaceqsierdhmdtn
equywxkqdkzplkefeehkuas
atxkfadugvpesmpipbjgmtk
rzrbjnddqwaadcfyjbppvbe 10
fshryuklnmcqxrtdhpdhtip
trsignlnsdujdqwqgkuyhms
ktjqdeysxrkfzzqorvlbbbz
rjjmisycdrntwstopkaxlzp
atllumovfaerkwpmbjbwftp
wlbyknyzprmanaqkgqoxamp
oqaqmfledwhcgwciltwnixg
sfwiezwtnmvfbdrfgpokylm
iyfeopbujisrnroihceodhf
kpxzavmmdmuhcllyvttmikc
