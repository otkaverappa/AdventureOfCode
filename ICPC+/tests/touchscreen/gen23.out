xpmqvtikeiwdetwztnnwtyy 75
wuddferqwlymuyiqruhhrjz 78
fibifhmjfkinoaesdsurbzu 82
juvkuenbchegnkzfimdugce 90
fwszpvajjkqojwtajirefso 92
wvzgesrebnznhfbkikqqzap 99
lqlvbibchsqyoliyprwtxqr 103
oheezgfolprvuykeguotkra 103
wlejizjilavgbgkfncynqjw 106
jakztvstapnbgjbqsliixgb 111
bavrsbrjisthwuupviquxsl 73
qzcqptbllcaywmdjwweqwxw 76
hjekjzyrhjstnsdmxrnpxcg 80
gzpvooyvnrrfwwijiuydrpa 83
ywhybbpbheiwnimtocppgar 84
ytwsedalkmpuvqvoxehttzw 87
fcblsmxqcytspovazyaauof 94
fykxebnrzlbzpcpsjtchgkt 98
goephyofgnubejzzyrsdpon 98
glbkbyjxfobqgnpzrpujfie 103
gsbzoyraqvgkhvonmvrvsyy 63
nytfyxfjrddglrmulxudkjt 63
ffuyamwbpmuuajxietqyeal 85
ryunimggtlasnpqogclafuz 85
ndgkmcyjtadeejagrogowlu 86
nteuicfccolqtqeyixkkcal 86
aptdmmagvuhulilnyitiqry 91
ahhoedodqrlcdoaoharnhan 92
brymsdirsimygkdpdhvjuov 94
ozulegwahoyviddbvnovvyz 96
juktupjtkqdnbaofpeliwrb 66
bhmkgthkdezckminuttwubv 75
scbxpcsibyznntwqfrhyfxc 79
bbeuayxvdmmtzqjjulfjtyi 80
uqcbbxajnxuvsvnzzhhycqx 85
vejblvycpqgrsftbsqxokud 85
nvkivzqectizthyrmkffjvq 86
xerjotjbksrtondazjphyhx 89
hwnxengbfkeczixilvjykcv 90
emronbqicawpirwuwctyohj 99
tnywzohaeurltkjqiclukop 75
qlphytwvwtvkfbswczpmeul 81
zbggnouvvapknyhzddjqgtc 82
bwiosschzvyzzutrkbgzmku 85
heuonjvjeuklbimlfxpykbj 86
szcjxmlhlopuddoybopnepu 86
duammggpyftkefxnqldkqia 87
gqoyeuupohnmjdvjhqculoe 92
xtmzoxnsalaofvtpwonvpjo 96
twzghxzivletkklpzxfkflg 102
tfsbgtkyfdtkvypcehvuxrl 74
mnmfbkchfkhvqyqjlcqwiap 83
tpnzsehmyytbxymucmrovxe 85
oarsimvpfchjjcvhkacuwcj 89
wwzlrruhzqzqlerdfmhgwkx 89
upgbxdlweaxkheqziutypej 90
zmyefhgsdcsovllckbdbpis 93
ktucqddhesqqquqpjeegvbf 100
vfaogrvrkmgjeksqksllqja 104
znjmybyypnnupxzyelizbgs 104
nnhrhfuqfciffvtvzmcyoxs 69
lmxwarlcsrmbrqzgivntgxc 71
jjqzewhbgwwfgybqujnyiil 75
fyofjuiqjxzceeywjbeqgck 80
tkjhmxijjxjfxatzsmovdgx 84
onvfyfweqhxrltypzxnedlh 86
bekjxcwnqugkkihlnhmbmbv 94
kpsziawkvivcwsngdkpljpe 96
edyibogvvgkbpelhpjirpmq 101
oyrsozwadazmdnuzlajmrpz 102
mvhayefupdpvzvtbjbgjyvh 72
yhhhkknhcggejvjtrqhfhga 73
dhznoivrzheisaunxqodvdu 77
rtzaaneiuhooowvgekrwhej 84
pmngiwmtuuaeenrnrztxkqi 85
nswvduqkhbyqjlxilzrnzgu 88
vkhjjtsdimbdrvvcfxcxiov 88
kwvgtgwclgdawnvnoftwxwr 92
njmhglfzxaiaanhwellzkju 95
fdeshifqgpkrxlnqdwhbdfw 97
gwuiucyrvznfnlxyfxqjkdi 82
aoabbzzwezqfxgvzovfrwre 93
dyubxqncgafaakntvklfnxx 93
kccpkfkihtuuvjgrbwdnbst 94
zjkncpktrqkkdngbrbtswhf 100
aeqsnbwvevqzsolotwcibvv 101
elwxdshkfjouwugebzyxwkd 102
pmqcrdzqfpqhsrgovfedeny 109
bxsvxwmuejopfqnympltcxr 113
plbsyektjvpiakrlrdbnzvt 113
rqkinoxdjxohwrxmoljarfn 84
gyrqemtjrlxjegjlxzohesc 87
froqrhxgpqdkdeupwvoxlaw 89
lfjiqgulsflkustgfdqhkqn 92
uylwqvxxichqsnmfhsrtolm 95
ftutkzimsawaqkxtwukftbd 96
vxiotwzfsuxdvcgnaiovcpq 96
vylyzaopvnxlgtlomhlrepa 105
utslacbqybbvsysrjvazpqo 107
ljasmfhmnkiahzehgyasjzf 111
fdcvmgnoednorzdduogdhvh 69
rmsbnikdqfqljmogcoxkvxr 80
heznjgkrnpqvojjnhbsagfi 86
agzzquwhbiznrxghnqqfkhk 88
ekqmgwtophjsurfyjkoutci 89
cqvwqbbafzysdaqsjujijjs 98
hdxgwrnzyqdtflitqljgekj 99
itktbwcvoyqghygjmxguggi 104
swtqgiaamqxijaljgkjynjq 113
cvvleaxpjepznxjgnzyinht 119
hluivqqconfnslfpdsbiwtg 76
lxplmdrqtldbvlddbdbhuih 77
wwbydeqvrnmdbpwtkhwlckq 81
qsngiicioyyerezzmuagars 83
fwoqsgtgivbsgpjlexyolfc 85
zgzaqvvykznysiybimhtudw 87
buitcsdodvldkegkaqiwpdo 97
azsklmrdpmniohlckjqqtuy 99
fpoxpyavxjuzbpvacwpcoyb 99
pdportyjaldkgwbkkqqzkbe 100
npbbbbcchtiojieurzylcgz 64
ztybfdumhorbmkxnuaddges 82
hhfoimrdzgecjhelrauflcl 83
xnpqulqiskywibcsldpdnci 83
hrqbybgkkbxpimfpjdeykic 86
htlfgpohxxpnhbemdmrhwao 86
gcvlibsarqbzyusgogojzzz 89
etduwruzicqjgzmfshvhriz 96
nyzigtrfsmgvtlphsfywtit 97
grgenaojgteudiodzggafbt 98
azwwocyrrbtvuwhlooqjxlu 70
sbokgeoqjdeureqibwnqbgx 81
trbuqkdzkfvjtahltjeodcx 81
obbetdkqsffthulwefzxgjj 89
zpgtbdyscctepyjsvluoyfi 89
qsnjbkwxdnzzdlddxkagdcq 90
qvzwleykxqxgthcrmrsmufp 94
zczcleebvactliswfktlgdp 96
ntnbcpqnhtdtbhjsqnwnead 101
ukvqydvpckyvobtgfxcmncx 109
umuupmkvwnnuaifgutlxeum 74
xqhgnysfxjvynzdplhazcrw 77
qzyssjmdgyxcplmlkzaecim 79
wnpmcddhatqijocollllanq 82
llvnpzwdjbiifuzvyrmwgme 85
qyxsacjyywnmaspjsjzcsoe 90
zrpbdcdakeiotrxtuqopmof 93
wsmlcmmyxxvlpvjsgmbptxq 101
ibswnslfkldclhnkflztlsj 107
kdfxclrqesowmaiqginiybw 107
ezwzhfuqmokoyvikapqugfv 66
tcjnzyvzyivnfwqcvnsrruq 85
tjlwaevtxsdznvomjybkjdy 90
cwshavsoovzdwzyxyjsnqrd 95
soqnijhymdstfnotdlvihuj 101
tdmzrmcrhoefqjajiuohhlx 101
vhzeekarqznsjuozhemxdhf 109
hssziksldhespnbpbehxlul 115
vuwxhhafafrczggqpwuxnfz 120
ynupvbauxxqwezwjccdeblm 122
wlbloegjmbgdqbiqikcvbjj 71
sjqegtnniykpdqbdtxgaytk 74
nyjhmtyighxhuowezgpddts 83
ivxpntcygcjnofmwntvxpjt 84
lhpbmpnilhicvepdadhstll 87
ryuyymajgjbmrwzfgzrxehz 90
xwfvtwgpsrqfixltjozpedx 93
abusaybbiezxlyiaptofamw 95
hcypfwippjlohyqsgcwttuu 98
usyqntliwocqocrsmxrmirw 100
ezimbjoruaceqsierdhmdtn 59
akmbzibxwsdvgvadwazegyy 79
kplvfidzihskttldnogcyao 80
atxkfadugvpesmpipbjgmtk 86
gsxhhekgbxncgymaisbnuyy 86
lmlipoeoytbvakyzuhrgams 87
tbqvhqxyqhmrgvvftnkqvgs 88
aeszkzzksdbpktmkkvxdduy 102
equywxkqdkzplkefeehkuas 103
nqdxosfxyxuigysmbqcdyar 105
trsignlnsdujdqwqgkuyhms 75
oqaqmfledwhcgwciltwnixg 78
ktjqdeysxrkfzzqorvlbbbz 81
iyfeopbujisrnroihceodhf 83
sfwiezwtnmvfbdrfgpokylm 86
wlbyknyzprmanaqkgqoxamp 89
fshryuklnmcqxrtdhpdhtip 90
atllumovfaerkwpmbjbwftp 94
rjjmisycdrntwstopkaxlzp 95
kpxzavmmdmuhcllyvttmikc 105
